dali/test/python/operator_2/test_warp.py

```diff
@@ -302,8 +303,8 @@ def get_data():
     for c in range(channels):
         assert out[0, 0, c] == c
 
-
-def test_extremely_large_data():
+@attr('slow')
+def slow_test_extremely_large_data():
```

Comment:
Strictly speaking, this test isn't slow. It takes 2s on A100 - unless you hit a slow node and then it can take 300s, but I don't think that getting rid of all tests that may hurt some machine's feelings is a good idea.